src/com/sap/piper/JenkinsJobController.groovy

```diff
@@ -0,0 +1,97 @@
+package com.sap.piper
+
+class JenkinsJobController implements Serializable {
+    def script
+    def timeout = 3600
+    String jenkinsUrl = "http://localhost:8080"
+    String jobName
+
+    //Trigger scanning of the multi branch builds
+    def buildJob() {
+        script.sh "curl -s -X POST ${jenkinsUrl}/job/${jobName}/build"
+    }
+
+    def getBuildUrl(String branch) {
+        return "${jenkinsUrl}/job/${jobName}/job/${URLEncoder.encode(branch, "UTF-8")}/lastBuild/"
```

Comment:
Is it also required to urlEncode the jobName? If so this needs to be done also elsewhere in this class/gScript.